css-images-4/Overview.bs

```diff
@@ -1701,7 +1701,17 @@ Conic Gradient Examples</h4>
 			width: 200px; height: 200px;
 		</pre>
 
-		<img src="images/conic5.png" alt="" >
+		You can also achieve the same effect using "longer" hue interpolation. This is more concise and makes it easy to switch to other color spaces.
+
+		<pre class=lang-css>
+			background: radial-gradient(closest-side, gray, transparent),
+			            conic-gradient(in hsl longer hue, red 0 100%);
+			transform: scaleX(-1);
```

Comment:
This is weird, we should just flip the image instead and go with a standard rainbow.

Reply:
You can always just reverse your color stops. That won't work for *this particular* rainbow example (you'll have to add some additional stops instead), but that's fine; using `longer hue` specifically to generate rainbows is something that's neat but not intentionally supported as a feature.